bigquery/README.md

```diff
@@ -56,8 +56,8 @@ async fn run(client: &Client, project_id: &str) {
         query: "SELECT * FROM dataset.table".to_string(),
         ..Default::default()
     };
-    let mut iter = client.query(project_id, request).await.unwrap();
-    while let Some(row) = iter.next::<Row>().await.unwrap() {
+    let mut iter = client.query::<Row>(project_id, request).await.unwrap();
+    while let Some(row) = iter.next().await.unwrap() {
```

Comment:
Compiler complained here to "remove the generic" from `iter.next::<Row>`. Meanwhile, the documentation for `client.query` itself includes `query::<Row>` as the correct type annotation.